starlight_help/astro.config.mjs

```diff
@@ -317,7 +317,7 @@ export default defineConfig({
                         "share-and-upload-files",
                         {
                             label: "Animated GIFs",
-                            link: "/animated-gifs-from-giphy",
+                            link: "/animated-gifs",
```

Comment:
You should add an entry in `redirects` too.

I think this commit should also be called "help: Rename article on animated GIF picker." -- it's not just updating links, it's also renaming the page.